src/agor/tools/dev_tooling.py

```
@@ -22,43 +22,37 @@
-
-    from .memory_manager import (
-        commit_to_memory_branch as _commit_to_memory_branch,
-        auto_commit_memory as _auto_commit_memory
-    )
+from .git_operations import (
+    get_current_timestamp,
+    get_file_timestamp,
+    get_precise_timestamp,
+    get_ntp_timestamp,
+    run_git_command,
+    safe_git_push,
```

Ruff (0.11.9)

`.git_operations.safe_git_push` imported but unused

Remove unused import: `.git_operations.safe_git_push`

(F401)